src/bentoml/_internal/monitoring/otlp.py

```diff
@@ -128,13 +135,23 @@ def __init__(
         self._will_export_schema = False
 
     def _init_logger(self) -> None:
+        from opentelemetry.sdk.resources import SERVICE_NAME
```

Comment:
How this works?

Reply:
This follows our change within our tracer API in the BentoMLContainer